vuu-ui/showcase/src/App.tsx

```diff
@@ -103,7 +105,8 @@ export const App = ({ stories }: AppProps) => {
             resizeable
             style={{ flexDirection: "column", flex: "1 1 auto" }}
           >
-            <Toolbar className="ShowcaseContentToolbar salt-density-high">
+            <Toolbar className={`ShowcaseContentToolbar salt-density-${density}`}>
```

Comment:
I don't think its a good idea to set density on the Content Toolbar, we just want it on the Content area below. The toolbar is part of the ShowCase application chrome , it shouldn't jump about as user changes density. At some point we might want to offer the ability to set both theme and density on the entire Showcase app, but then we should offer an application level theme/density switcher on the very top toolbar. That can come later - we would have to make sure the Nav Tree supports all densitied